src/bp/ui-studio/src/web/components/Layout/StatusBar/TrainingStatus/SingleLang.tsx

```diff
@@ -63,43 +62,46 @@ const TrainingStatusComponent: FC<Props> = (props: Props) => {
     e.preventDefault()
     onCanceling()
     try {
-      await axios.post(`${BASE_NLU_URL}/train/delete`)
+      await axios.post(`${BASE_NLU_URL}/train/delete/${trainSession.language}`)
     } catch (err) {
       console.error('cannot cancel training')
     } finally {
       setLoading(false)
     }
   }
 
-  if (status === null) {
+  if (!status) {
```

Comment:
I think there was a reason why we checked for null like a race condition or a unhandled status but that might be useless now.

Reply:
This is possible, but I don't see why this condition can't included undefined and empty string.

If there's no status, there's no rendering